### DISCUSSION TITLE
Vim keybindings in Dax Studio

### DISCUSSION THREAD

**Discussion by fhestvang** [DaxStudio/DaxStudio#1138] [Ideas]
Hi

I haven't been able to find any discussions around introducing classic keybindings in Dax Studio? Aren't the community using such in their other IDE's?

All the best
Frederik

**Comment by dgosbell** [comment]
There have been other asks for custom key bindings. You can change some key bindings under File > Options. But we are a little constrained in what we can change easily as many of the navigation keys are hard coded into the edit control we are using https://github.com/icsharpcode/AvalonEdit But from the little I know of vim it would require implementing different command modes in order to support all its key bindings and I do not think there is a lot of demand for that. (this is the first time anyone has mentioned vim keybindings)

**Comment by fhestvang** [comment]
Hi @dgosbell. 

Thanks for the quick response. Last sentence is dealbreaking in discussing this further - but thanks for the elaborate explanation. I'll wait untill DS is available as an extension within VSCode, haha. 😄 

Best
Frederik
